strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/LayoutFileSystemProvider.java

```diff
@@ -372,4 +376,8 @@ public void undelete(RepositoryPath path)
         return getLayoutProvider().getRepositoryFileAttributes(repositoryRelativePath, attributeTypes);
     }
     
+    protected abstract void deleteMetadata(RepositoryPath repositoryPath)
```

Comment:
we can do this method just empty instead of `abstrect` for not to override it every time.